src/replication.c

```diff
@@ -3738,7 +3738,7 @@ void replicationCron(void) {
     if (server.masterhost && server.repl_state == REPL_STATE_TRANSFER &&
         (time(NULL)-server.repl_transfer_lastio) > server.repl_timeout)
     {
-        serverLog(LL_WARNING,"Timeout receiving bulk data from MASTER... If the problem persists try to set the 'repl-timeout' parameter in redis.conf to a larger value.");
+        serverLog(LL_WARNING,"Timeout receiving bulk data from MASTER... If the problem persists try to set the 'repl-timeout' parameter config to a larger value.");
```

Comment:
Could you pls give a reason why we do not follow the other changes as placeholderkv.conf?

Reply:
My vote here was that usage of "parameter in redis.conf" is really weird, since it's also possible you could be setting at startup.

Reply:
```suggestion
        serverLog(LL_WARNING,"Timeout receiving bulk data from MASTER... If the problem persists try to set the 'repl-timeout' config to a larger value.");
```